ENIGMAsystem/SHELL/Bridges/General/Win32-Direct3D11.cpp

```diff
@@ -386,8 +215,8 @@ namespace enigma_user
   }
 
   void screen_refresh() {
-      window_set_caption(room_caption);
-      enigma::update_mouse_variables();
+    window_set_caption(room_caption);
```

Comment:
Please don't do this here...

Reply:
Haha, you mean the caption? That was polygonz who started it I believe, anyway it has proliferated through all of the bridges now, so it must be dealt with in a separate pull request. I can't do anything about it right now.

Here, I actually found proof it was polygonz:
https://github.com/enigma-dev/enigma-dev/issues/163#issuecomment-18440238

Reply:
Cool beans, I've filed #1336 for it.